### ISSUE TITLE
PHP 7.4 error curly braces deprecated in require_once() (and Update to markdown 1.9)

### ISSUE THREAD

**Issue by jenlampton** [backdrop-contrib/markdown#4]
In PHP 7.4, I see the following error on my status report:
```
Deprecated function: Array and string offset access syntax with curly braces is deprecated in require_once() (line 10 of backdrop/modules/contrib/markdown/includes/Markdown.inc.php).
```

It looks like we may need to update to the 1.9 release of markdown.

See related: 
* https://github.com/michelf/php-markdown/issues/321
* https://tracker.moodle.org/browse/MDL-66964

**Comment by jenlampton** [comment]
I discovered a syntax error in `module_load_include()` that seems to fix this issue. The update to 1.9.0 wasn't necessary, so I separated that out into a different issue.

**Comment by jenlampton** [comment]
Well, I was wrong, we do need to update the library to avoid this error. I've included that commit on this PR and closed that other issue.

**Comment by bugfolder** [comment]
Maybe time for a merge of the PR and new release?

**Comment by alanmels** [comment]
The module has not been touched for 6 years already. Are two of the maintainers still interested in the module?

-     Chris Herberte (https://github.com/chrisherberte)
-     Fredrik Jonsson (https://github.com/frjo)

**Comment by bugfolder** [comment]
@alanmels, do you want to adopt the module if it's been abandoned? (I will if you don't want to.)

[There's an official process for determining if a module has been abandoned.](https://docs.backdropcms.org/documentation/adopt-a-project) I'll start that process by filing the issue.

**Comment by alanmels** [comment]
@bugfolder, unfortunately no time, so please go ahead take over the maintainership of the module as the current ones do not seem to be reacting at all.

**Comment by alanmels** [comment]
I've updated the library to 2.0.0 See new PR https://github.com/backdrop-contrib/markdown/pull/10/files

**Comment by bugfolder** [comment]
Merged the update to 2.0.0. Thanks, @alanmels!
